packages/firestore/package.json

```diff
@@ -80,6 +79,7 @@
   },
   "devDependencies": {
     "@firebase/app": "0.6.14",
+    "@microsoft/api-extractor": "7.13.1",
```

Comment:
```suggestion
```

Currently we are using a fork of api-extractor that supports multiple entry points and it's already part of the root package.json.